src/tests/components/ForecastDetails.test.js

```diff
@@ -0,0 +1,29 @@
+import React from "react";
+import { render, screen } from "@testing-library/react";
+import renderer from "react-test-renderer";
+import ForecastDetails from "../../components/ForecastDetails";
+
+describe("ForecastDetails", () => {
+  const validProps = {
+    forecast: {
+      date: 1525046400000,
+      temperature: {
+        min: 4,
+        max: 11,
+      },
+      humidity: 30,
+      wind: {
+        speed: 13,
+        direction: "s",
+      },
+    },
+  };
+
+  it("renders correctly", () => {
```

Comment:
It would be good to add a lot more assertions here to check all the values passed in via the props are being rendered as expected